source/style-guide/style/voice-and-tone.txt

```diff
@@ -22,7 +22,8 @@ No matter where our words land, they should sound like us.
 Voice and Tone Attributes
 -------------------------
 
-The following table shows attributes for voice and tone:
+The following table shows attributes for voice and tone. These can be
+range in intensity. While our words should always use these basic attributes, they can also use amplified attributes when necessary.
```

Comment:
**questions:** 

1. Should we define V&T "attributes"?
2. Are we clear what we mean by intensity here? Should we add a clarifying sentence? 
3. What are the differences between _basic_ and _amplified_?

**nit:** Line wrap.

Reply:
Based on what was already here, I sort of struggled with how to frame this. Reworked.